### DISCUSSION TITLE
Generate a single value

### DISCUSSION THREAD

**Discussion by denizdogan** [dubzzz/fast-check#4942] [Q&A] [answered]
Is there any built-in way to generate a single value from an Arbitrary?

I use this utility, but I feel like there should exist something like this already.

```ts
import * as fc from "fast-check"

const single = <T>(arb: fc.Arbitrary<T>) => fc.sample(arb, 1)[0]
```

**Comment by dubzzz** [comment, marked as answer]
At the moment it's the recommended way. There is no built-in function provided to generate 'random' values outside of a property.

To generate random values and drop the bias applied by fast-check you may want to add the `.noBias()` on your arbitraries. Bias being more frequent for the very first generated values the `[0]` might be highly impacted by it. More at https://fast-check.dev/docs/core-blocks/arbitraries/combiners/any/#nobias
